makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
```

Comment:
@nhazekam , the logic here is that if the user does not set up the `--umbrella-spec`, it is not necessary to set up the umbrella wrapper.
That means, if the user only sets `--umbrella-binary` without setting `--umbrella-spec`, we do nothing.